### PR TITLE
Spell "Connect" @ `Session.connect()` docstring

### PR DESCRIPTION
**Pull request by donnerhacke**
##### SUMMARY
There is a spelling error in the description.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: docs/changelog-fragments/474.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed spelling in Session.connect -- by :user:`donnerhacke`
```
Comment (webknjaz) [inline docs/changelog-fragments/474.doc.rst]:
```suggestion
Fixed spelling of "Connect" in the ``Session.connect()``
docstring -- by :user:`donnerhacke`
```

File: pyproject.toml [hunk 2]
```diff
@@ -22,6 +22,7 @@ src = ["src/**/*.pyx"]
 #ANSIBLE_PYLIBSSH_TRACING = "1"
 #CFLAGS = "-DCYTHON_TRACE=1 ${CFLAGS}"
 LDFLAGS = "-lssh ${LDFLAGS}"
+CFLAGS="-I /usr/local/include ${CFLAGS}"
```
Comment (webknjaz) [inline pyproject.toml]: This shouldn't be a part of a spelling PR.

Comment (donnerhacke) [reply pyproject.toml]: Of course not. A branching reference error

Comment (webknjaz) [reply pyproject.toml]: Looks like fixing this is the only blocker here.

**Pull request merged**